test/1.4/provisional/T_bare.dml

```diff
@@ -0,0 +1,17 @@
+/*
+  © 2024 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+
+dml 1.4;
+
+/// COMPILE-ONLY
+
+provisional;
```

Comment:
huh. I guess that makes sense. I just checked, Haskell permits a bare `{-# LANGUAGE #-}` too.

Reply:
There is no intended use case, this is just a grammatical edge case; I added the test just to keep an unimportant corner stable.